rb/lib/selenium/webdriver.rb

```diff
@@ -57,8 +57,10 @@ def self.root
     #
     # @return [Driver]
     #
-    # @see Selenium::WebDriver::Remote::Bridge
+    # @see Selenium::WebDriver::Remote::OSSBridge
```

Comment:
That would have been my name for it as well, but I copied how Simon named his subclassing in Java. "OSS" and "W3C."